lib/subscriptionClient.ts

```diff
@@ -153,33 +118,28 @@ export class SubscriptionClient extends Client {
    * @param {number} maxMessageCount        The maximum message count. Must be a value greater than 0.
    * @param {number} [maxWaitTimeInSeconds] The maximum wait time in seconds for which the Receiver
    * should wait to receiver the said amount of messages. If not provided, it defaults to 60 seconds.
-   * @param {ReceiveOptions} [options]      Options for how you'd like to connect.
    *
    * @returns {Promise<Message[]>} A promise that resolves with an array of Message objects.
    */
   async receiveBatch(maxMessageCount: number, maxWaitTimeInSeconds?: number): Promise<Message[]> {
     if (!this._context.batchingReceiver ||
-      (this._context.batchingReceiver && !this._context.batchingReceiver.isOpen()) ||
-      (this._context.batchingReceiver && !this._context.batchingReceiver.isReceivingMessages)) {
+      !this._context.batchingReceiver.isOpen() ||
+      !this._context.batchingReceiver.isReceivingMessages) {
       const options: ReceiveOptions = {
-        maxConcurrentCalls: this.maxConcurrentCalls,
+        maxConcurrentCalls: 0,
```

Comment:
I noticed the `queueClient` also had the `maxConcurrentCalls` hard coded to `0` and wanted to verify that it should also be `0` here?

Reply:
Yup this should be 0 as well. In batch mode, the maxConcurrentCalls translates to the number of messages the customer wants to receive.